aten/src/ATen/native/native_functions.yaml

```diff
@@ -1987,6 +1987,9 @@
 - func: rot90(Tensor self, int k=1, int[] dims=[0,1]) -> Tensor
   variants: function, method
 
+- func: trapz(Tensor y, Tensor x, *, int dim=-1) -> Tensor
```

Comment:
This is missing documentation. You should add an entry here: https://github.com/pytorch/pytorch/blob/master/torch/_torch_docs.py and here: https://github.com/pytorch/pytorch/blob/master/torch/_tensor_docs.py.

cd into `docs/` and run `make html` to generate the docs for viewing on a local machine.

Reply:
I will add something to `_torch_docs`, but this is only a function, not a method, so I don't think it is needed in `_tensor_docs`.

Reply:
Yeah you are right, I get confused as to what the default is.